src/Umbraco.Web.UI/config/tinyMceConfig.config

```diff
@@ -5,204 +5,228 @@
+      <name>Strikethrough</name>
       <icon>images/editor/strikethrough.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="strikethrough">strikethrough</tinyMceCommand>
       <priority>24</priority>
     </command>
-
     <command>
-      <umbracoAlias>JustifyLeft</umbracoAlias>
+      <umbracoAlias>justifyleft</umbracoAlias>
+      <name>Justify Left</name>
       <icon>images/editor/justifyleft.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="alignleft">justifyleft</tinyMceCommand>
       <priority>31</priority>
     </command>
     <command>
-      <umbracoAlias>JustifyCenter</umbracoAlias>
+      <umbracoAlias>justifycenter</umbracoAlias>
+      <name>Justify Center</name>
       <icon>images/editor/justifycenter.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="aligncenter">justifycenter</tinyMceCommand>
       <priority>32</priority>
     </command>
     <command>
-      <umbracoAlias>JustifyRight</umbracoAlias>
+      <umbracoAlias>justifyright</umbracoAlias>
+      <name>Justify Right</name>
       <icon>images/editor/justifyright.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="alignright">justifyright</tinyMceCommand>
       <priority>33</priority>
     </command>
     <command>
-      <umbracoAlias>JustifyFull</umbracoAlias>
+      <umbracoAlias>justifyfull</umbracoAlias>
+      <name>Justify Full</name>
       <icon>images/editor/justifyfull.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="alignjustify">alignjustify</tinyMceCommand>
       <priority>34</priority>
     </command>
-
     <command>
       <umbracoAlias>bullist</umbracoAlias>
+      <name>Bullet list</name>
       <icon>images/editor/bullist.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="bullist">bullist</tinyMceCommand>
       <priority>41</priority>
     </command>
     <command>
       <umbracoAlias>numlist</umbracoAlias>
+      <name>Numbered list</name>
       <icon>images/editor/numlist.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="numlist">numlist</tinyMceCommand>
       <priority>42</priority>
     </command>
     <command>
-      <umbracoAlias>Outdent</umbracoAlias>
+      <umbracoAlias>outdent</umbracoAlias>
+      <name>Decrease indent</name>
       <icon>images/editor/outdent.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="outdent">outdent</tinyMceCommand>
       <priority>43</priority>
     </command>
     <command>
-      <umbracoAlias>Indent</umbracoAlias>
+      <umbracoAlias>indent</umbracoAlias>
+      <name>Increase indent</name>
       <icon>images/editor/indent.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="indent">indent</tinyMceCommand>
       <priority>44</priority>
     </command>
-
     <command>
       <umbracoAlias>mceLink</umbracoAlias>

```

Comment:
Not sure if there is a reason why they use "mce", e.g. "mceLink" instead of just "link", while using "unlink" like here http://archive.tinymce.com/wiki.php/TinyMCE3x:Buttons/controls